user/super/com/google/gwt/emul/java/util/stream/Collectors.java

```diff
@@ -170,6 +170,34 @@ public static Collector<CharSequence,?,String> joining(CharSequence delimiter) {
         downstream.finisher());
   }
 
+  public static <T, U, A, R> Collector<T, ?, R> flatMapping(Function<? super T,? extends Stream<?
+      extends U>> mapper, Collector<? super U, A, R> downstream) {
+    return new CollectorImpl<>(
+        downstream.supplier(),
+        (A a, T t) -> {
+            mapper.apply(t).forEach(u -> {
+              downstream.accumulator().accept(a, u);
```

Comment:
from Javadoc

> (If a mapped stream is {@code null} an empty stream is used, instead.)

Reply:
While, you're right, that's what the Javadoc says... whats the point of treating null as empty here but forcing null to throw elsewhere. I don't care for the inconsistency.

Thanks for pointing this out, I'll make the fix and test for it.